lib/jekyll/commands/unpublish.rb

```diff
@@ -53,6 +53,10 @@ def from
       def to
         "_drafts/#{params.name}"
       end
+
+      def front_matter(data)
+        data.reject { |x| x == "date" }
```

Comment:
`data` is traditionally a Hash instance. Unlike `Array#reject`, `Hash#reject` produces two block parameters: `key` and `value`:
```suggestion
        data.reject { |key, _value| key == "date" }
```

Reply:
Oh, yeah, that needs to be fixed.